.code-samples.meilisearch.yaml

```diff
@@ -1147,6 +1206,15 @@ getting_started_faceting: |-
     .set_faceting(&faceting)
     .await
     .unwrap();
+getting_started_pagination: |-
+  let pagination = PaginationSetting {max_total_hits:500};
+
+  let task: TaskInfo = client
+    .index("books")
+    .set_pagination(pagination)
+    .await
+    .unwrap();
+  client.index("movies").updatePagination({ maxTotalHits: 500 })
```

Comment:
Shouldn't it be `update_pagination`? I couldn't find this method in the SDK.

Reply:
I forgot to remove it!